## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 467598`
Commit: [`e71559b1d50f20a08978f25c309dcaca0baf21ce`](https://github.com/NixOS/nixpkgs/commit/e71559b1d50f20a08978f25c309dcaca0baf21ce) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/e71559b1d50f20a08978f25c309dcaca0baf21ce..pull/467598/head))
Merge: [`cbcb61b75447dffced3c5a5f94143a774ac7e529`](https://github.com/NixOS/nixpkgs/commit/cbcb61b75447dffced3c5a5f94143a774ac7e529)

Logs: https://github.com/alejo7797/nixpkgs-review-gha/actions/runs/19915088842


---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>daikhan</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 59 packages built:</summary>
  <ul>
    <li>authenticator</li>
    <li>cobang</li>
    <li>dorion</li>
    <li>footage</li>
    <li>fractal</li>
    <li>gnome-contacts</li>
    <li>gnome-decoder</li>
    <li>gst_all_1.gst-plugins-rs</li>
    <li>gst_all_1.gst-plugins-rs.dev</li>
    <li>mopidy-bandcamp (mopidyPackages.mopidy-bandcamp)</li>
    <li>mopidy-bandcamp.dist (mopidyPackages.mopidy-bandcamp.dist)</li>
    <li>mopidy-iris (mopidyPackages.mopidy-iris)</li>
    <li>mopidy-iris.dist (mopidyPackages.mopidy-iris.dist)</li>
    <li>mopidy-jellyfin (mopidyPackages.mopidy-jellyfin)</li>
    <li>mopidy-jellyfin.dist (mopidyPackages.mopidy-jellyfin.dist)</li>
    <li>mopidy-listenbrainz (mopidyPackages.mopidy-listenbrainz)</li>
    <li>mopidy-listenbrainz.dist (mopidyPackages.mopidy-listenbrainz.dist)</li>
    <li>mopidy-local (mopidyPackages.mopidy-local)</li>
    <li>mopidy-local.dist (mopidyPackages.mopidy-local.dist)</li>
    <li>mopidy-mopify (mopidyPackages.mopidy-mopify)</li>
    <li>mopidy-mopify.dist (mopidyPackages.mopidy-mopify.dist)</li>
    <li>mopidy-mpd (mopidyPackages.mopidy-mpd)</li>
    <li>mopidy-mpd.dist (mopidyPackages.mopidy-mpd.dist)</li>
    <li>mopidy-mpris (mopidyPackages.mopidy-mpris)</li>
    <li>mopidy-mpris.dist (mopidyPackages.mopidy-mpris.dist)</li>
    <li>mopidy-muse (mopidyPackages.mopidy-muse)</li>
    <li>mopidy-muse.dist (mopidyPackages.mopidy-muse.dist)</li>
    <li>mopidy-musicbox-webclient (mopidyPackages.mopidy-musicbox-webclient)</li>
    <li>mopidy-musicbox-webclient.dist (mopidyPackages.mopidy-musicbox-webclient.dist)</li>
    <li>mopidy-notify (mopidyPackages.mopidy-notify)</li>
    <li>mopidy-notify.dist (mopidyPackages.mopidy-notify.dist)</li>
    <li>mopidy-podcast (mopidyPackages.mopidy-podcast)</li>
    <li>mopidy-podcast.dist (mopidyPackages.mopidy-podcast.dist)</li>
    <li>mopidy-scrobbler (mopidyPackages.mopidy-scrobbler)</li>
    <li>mopidy-scrobbler.dist (mopidyPackages.mopidy-scrobbler.dist)</li>
    <li>mopidy-somafm (mopidyPackages.mopidy-somafm)</li>
    <li>mopidy-somafm.dist (mopidyPackages.mopidy-somafm.dist)</li>
    <li>mopidy-soundcloud (mopidyPackages.mopidy-soundcloud)</li>
    <li>mopidy-soundcloud.dist (mopidyPackages.mopidy-soundcloud.dist)</li>
    <li>mopidy-spotify (mopidyPackages.mopidy-spotify)</li>
    <li>mopidy-spotify.dist (mopidyPackages.mopidy-spotify.dist)</li>
    <li>mopidy-subidy (mopidyPackages.mopidy-subidy)</li>
    <li>mopidy-subidy.dist (mopidyPackages.mopidy-subidy.dist)</li>
    <li>mopidy-tidal (mopidyPackages.mopidy-tidal)</li>
    <li>mopidy-tidal.dist (mopidyPackages.mopidy-tidal.dist)</li>
    <li>mopidy-tunein (mopidyPackages.mopidy-tunein)</li>
    <li>mopidy-tunein.dist (mopidyPackages.mopidy-tunein.dist)</li>
    <li>mopidy-youtube (mopidyPackages.mopidy-youtube)</li>
    <li>mopidy-youtube.dist (mopidyPackages.mopidy-youtube.dist)</li>
    <li>mopidy-ytmusic (mopidyPackages.mopidy-ytmusic)</li>
    <li>mopidy-ytmusic.dist (mopidyPackages.mopidy-ytmusic.dist)</li>
    <li>muzika</li>
    <li>pantheon.elementary-camera</li>
    <li>pantheon.elementary-videos</li>
    <li>recordbox</li>
    <li>showtime</li>
    <li>snapshot</li>
    <li>strawberry</li>
    <li>warp</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>daikhan</summary>
<pre>C linker for the host machine: gcc ld.bfd 2.44
Vala compiler for the host machine: valac (valac 0.56.18)
Host machine cpu family: x86_64
Host machine cpu: x86_64
Library m found: YES
Found pkg-config: YES (/nix/store/8vdiwpbh0g4avsd6x5v4s0di32vcl3dp-pkg-config-wrapper-0.29.2/bin/pkg-config) 0.29.2
Run-time dependency libadwaita-1 found: YES 1.8.1
Run-time dependency gdk-pixbuf-2.0 found: YES 2.44.4
Run-time dependency gio-2.0 found: YES 2.86.1
Run-time dependency gstreamer-1.0 found: YES 1.26.5
Run-time dependency gstreamer-audio-1.0 found: YES 1.26.5
Run-time dependency gstreamer-tag-1.0 found: YES 1.26.5
Run-time dependency libxxhash found: YES 0.8.3
Did not find CMake &#x27;cmake&#x27;
Found CMake: NO
Run-time dependency gstgtk4 found: NO (tried pkgconfig)

meson.build:36:0: ERROR: Dependency &quot;gstgtk4&quot; not found, tried pkgconfig

A full log can be found at /build/source/build/meson-logs/meson-log.txt</pre>
</details>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>daikhan</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 59 packages built:</summary>
  <ul>
    <li>authenticator</li>
    <li>cobang</li>
    <li>dorion</li>
    <li>footage</li>
    <li>fractal</li>
    <li>gnome-contacts</li>
    <li>gnome-decoder</li>
    <li>gst_all_1.gst-plugins-rs</li>
    <li>gst_all_1.gst-plugins-rs.dev</li>
    <li>mopidy-bandcamp (mopidyPackages.mopidy-bandcamp)</li>
    <li>mopidy-bandcamp.dist (mopidyPackages.mopidy-bandcamp.dist)</li>
    <li>mopidy-iris (mopidyPackages.mopidy-iris)</li>
    <li>mopidy-iris.dist (mopidyPackages.mopidy-iris.dist)</li>
    <li>mopidy-jellyfin (mopidyPackages.mopidy-jellyfin)</li>
    <li>mopidy-jellyfin.dist (mopidyPackages.mopidy-jellyfin.dist)</li>
    <li>mopidy-listenbrainz (mopidyPackages.mopidy-listenbrainz)</li>
    <li>mopidy-listenbrainz.dist (mopidyPackages.mopidy-listenbrainz.dist)</li>
    <li>mopidy-local (mopidyPackages.mopidy-local)</li>
    <li>mopidy-local.dist (mopidyPackages.mopidy-local.dist)</li>
    <li>mopidy-mopify (mopidyPackages.mopidy-mopify)</li>
    <li>mopidy-mopify.dist (mopidyPackages.mopidy-mopify.dist)</li>
    <li>mopidy-mpd (mopidyPackages.mopidy-mpd)</li>
    <li>mopidy-mpd.dist (mopidyPackages.mopidy-mpd.dist)</li>
    <li>mopidy-mpris (mopidyPackages.mopidy-mpris)</li>
    <li>mopidy-mpris.dist (mopidyPackages.mopidy-mpris.dist)</li>
    <li>mopidy-muse (mopidyPackages.mopidy-muse)</li>
    <li>mopidy-muse.dist (mopidyPackages.mopidy-muse.dist)</li>
    <li>mopidy-musicbox-webclient (mopidyPackages.mopidy-musicbox-webclient)</li>
    <li>mopidy-musicbox-webclient.dist (mopidyPackages.mopidy-musicbox-webclient.dist)</li>
    <li>mopidy-notify (mopidyPackages.mopidy-notify)</li>
    <li>mopidy-notify.dist (mopidyPackages.mopidy-notify.dist)</li>
    <li>mopidy-podcast (mopidyPackages.mopidy-podcast)</li>
    <li>mopidy-podcast.dist (mopidyPackages.mopidy-podcast.dist)</li>
    <li>mopidy-scrobbler (mopidyPackages.mopidy-scrobbler)</li>
    <li>mopidy-scrobbler.dist (mopidyPackages.mopidy-scrobbler.dist)</li>
    <li>mopidy-somafm (mopidyPackages.mopidy-somafm)</li>
    <li>mopidy-somafm.dist (mopidyPackages.mopidy-somafm.dist)</li>
    <li>mopidy-soundcloud (mopidyPackages.mopidy-soundcloud)</li>
    <li>mopidy-soundcloud.dist (mopidyPackages.mopidy-soundcloud.dist)</li>
    <li>mopidy-spotify (mopidyPackages.mopidy-spotify)</li>
    <li>mopidy-spotify.dist (mopidyPackages.mopidy-spotify.dist)</li>
    <li>mopidy-subidy (mopidyPackages.mopidy-subidy)</li>
    <li>mopidy-subidy.dist (mopidyPackages.mopidy-subidy.dist)</li>
    <li>mopidy-tidal (mopidyPackages.mopidy-tidal)</li>
    <li>mopidy-tidal.dist (mopidyPackages.mopidy-tidal.dist)</li>
    <li>mopidy-tunein (mopidyPackages.mopidy-tunein)</li>
    <li>mopidy-tunein.dist (mopidyPackages.mopidy-tunein.dist)</li>
    <li>mopidy-youtube (mopidyPackages.mopidy-youtube)</li>
    <li>mopidy-youtube.dist (mopidyPackages.mopidy-youtube.dist)</li>
    <li>mopidy-ytmusic (mopidyPackages.mopidy-ytmusic)</li>
    <li>mopidy-ytmusic.dist (mopidyPackages.mopidy-ytmusic.dist)</li>
    <li>muzika</li>
    <li>pantheon.elementary-camera</li>
    <li>pantheon.elementary-videos</li>
    <li>recordbox</li>
    <li>showtime</li>
    <li>snapshot</li>
    <li>strawberry</li>
    <li>warp</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>daikhan</summary>
<pre>C linker for the host machine: gcc ld.bfd 2.44
Vala compiler for the host machine: valac (valac 0.56.18)
Host machine cpu family: aarch64
Host machine cpu: aarch64
Library m found: YES
Found pkg-config: YES (/nix/store/lbiigi8qbp7mzf1lpr7p982l1kyf01ql-pkg-config-wrapper-0.29.2/bin/pkg-config) 0.29.2
Run-time dependency libadwaita-1 found: YES 1.8.1
Run-time dependency gdk-pixbuf-2.0 found: YES 2.44.4
Run-time dependency gio-2.0 found: YES 2.86.1
Run-time dependency gstreamer-1.0 found: YES 1.26.5
Run-time dependency gstreamer-audio-1.0 found: YES 1.26.5
Run-time dependency gstreamer-tag-1.0 found: YES 1.26.5
Run-time dependency libxxhash found: YES 0.8.3
Did not find CMake &#x27;cmake&#x27;
Found CMake: NO
Run-time dependency gstgtk4 found: NO (tried pkgconfig)

meson.build:36:0: ERROR: Dependency &quot;gstgtk4&quot; not found, tried pkgconfig

A full log can be found at /build/source/build/meson-logs/meson-log.txt</pre>
</details>
</details>

---
### `x86_64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>footage</li>
    <li>muzika</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 45 packages built:</summary>
  <ul>
    <li>gst_all_1.gst-plugins-rs</li>
    <li>gst_all_1.gst-plugins-rs.dev</li>
    <li>mopidy-bandcamp (mopidyPackages.mopidy-bandcamp)</li>
    <li>mopidy-bandcamp.dist (mopidyPackages.mopidy-bandcamp.dist)</li>
    <li>mopidy-iris (mopidyPackages.mopidy-iris)</li>
    <li>mopidy-iris.dist (mopidyPackages.mopidy-iris.dist)</li>
    <li>mopidy-jellyfin (mopidyPackages.mopidy-jellyfin)</li>
    <li>mopidy-jellyfin.dist (mopidyPackages.mopidy-jellyfin.dist)</li>
    <li>mopidy-listenbrainz (mopidyPackages.mopidy-listenbrainz)</li>
    <li>mopidy-listenbrainz.dist (mopidyPackages.mopidy-listenbrainz.dist)</li>
    <li>mopidy-local (mopidyPackages.mopidy-local)</li>
    <li>mopidy-local.dist (mopidyPackages.mopidy-local.dist)</li>
    <li>mopidy-mopify (mopidyPackages.mopidy-mopify)</li>
    <li>mopidy-mopify.dist (mopidyPackages.mopidy-mopify.dist)</li>
    <li>mopidy-mpd (mopidyPackages.mopidy-mpd)</li>
    <li>mopidy-mpd.dist (mopidyPackages.mopidy-mpd.dist)</li>
    <li>mopidy-mpris (mopidyPackages.mopidy-mpris)</li>
    <li>mopidy-mpris.dist (mopidyPackages.mopidy-mpris.dist)</li>
    <li>mopidy-muse (mopidyPackages.mopidy-muse)</li>
    <li>mopidy-muse.dist (mopidyPackages.mopidy-muse.dist)</li>
    <li>mopidy-musicbox-webclient (mopidyPackages.mopidy-musicbox-webclient)</li>
    <li>mopidy-musicbox-webclient.dist (mopidyPackages.mopidy-musicbox-webclient.dist)</li>
    <li>mopidy-notify (mopidyPackages.mopidy-notify)</li>
    <li>mopidy-notify.dist (mopidyPackages.mopidy-notify.dist)</li>
    <li>mopidy-podcast (mopidyPackages.mopidy-podcast)</li>
    <li>mopidy-podcast.dist (mopidyPackages.mopidy-podcast.dist)</li>
    <li>mopidy-scrobbler (mopidyPackages.mopidy-scrobbler)</li>
    <li>mopidy-scrobbler.dist (mopidyPackages.mopidy-scrobbler.dist)</li>
    <li>mopidy-somafm (mopidyPackages.mopidy-somafm)</li>
    <li>mopidy-somafm.dist (mopidyPackages.mopidy-somafm.dist)</li>
    <li>mopidy-soundcloud (mopidyPackages.mopidy-soundcloud)</li>
    <li>mopidy-soundcloud.dist (mopidyPackages.mopidy-soundcloud.dist)</li>
    <li>mopidy-spotify (mopidyPackages.mopidy-spotify)</li>
    <li>mopidy-spotify.dist (mopidyPackages.mopidy-spotify.dist)</li>
    <li>mopidy-subidy (mopidyPackages.mopidy-subidy)</li>
    <li>mopidy-subidy.dist (mopidyPackages.mopidy-subidy.dist)</li>
    <li>mopidy-tidal (mopidyPackages.mopidy-tidal)</li>
    <li>mopidy-tidal.dist (mopidyPackages.mopidy-tidal.dist)</li>
    <li>mopidy-tunein (mopidyPackages.mopidy-tunein)</li>
    <li>mopidy-tunein.dist (mopidyPackages.mopidy-tunein.dist)</li>
    <li>mopidy-youtube (mopidyPackages.mopidy-youtube)</li>
    <li>mopidy-youtube.dist (mopidyPackages.mopidy-youtube.dist)</li>
    <li>mopidy-ytmusic (mopidyPackages.mopidy-ytmusic)</li>
    <li>mopidy-ytmusic.dist (mopidyPackages.mopidy-ytmusic.dist)</li>
    <li>showtime</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>footage</summary>
<pre>      - /nix/store/i1hq9jj7gclzqqbz7v7a2wmvb9z6s961-libadwaita-1.8.1-dev/lib/pkgconfig
      - /nix/store/dysr5h69s3adfxy8yzqyflw3wwhk8xn2-a52dec-0.8.0/lib/pkgconfig
      - /nix/store/nip51cbrgnd9rwcr7bsk0z4fpb8wm3y7-fdk-aac-2.0.3-dev/lib/pkgconfig
      - /nix/store/5jn79m47i62srkzw58yjf344ixbvhrqb-ffmpeg-8.0-dev/lib/pkgconfig
      - /nix/store/qxx7yvj3h033v6jpsdisy3wsqw7axqx0-x264-0-unstable-2025-01-03-dev/lib/pkgconfig
      - /nix/store/fyv9dqj7wg4nrqmb0mbzbn092vsr9qf9-x265-4.1-dev/lib/pkgconfig
      - /nix/store/9lh6l0hq6qm5cr4cvi9mjcn15w3cnjxa-vo-aacenc-0.1.3/lib/pkgconfig
      - /nix/store/732243l5p3qrpnnmh3hyn5ggbqcqdfcj-svt-av1-3.1.2/lib/pkgconfig
      - /nix/store/inr6ia2ynz2zri9hmcwkb2waxsah5q93-libmpeg2-0.5.1/lib/pkgconfig
      - /nix/store/hqyyrcwqbpsbijflgx1kmkka4ykjdx4z-gst-plugins-base-1.26.5-dev/lib/pkgconfig
      - /nix/store/prpr267fwjwnbk6vqaskl8k65v3s55by-gstreamer-1.26.5-dev/lib/pkgconfig
      - /nix/store/14wd84jr2wwfmvrkz9ncw953pvkn3wvp-gst-plugins-bad-1.26.5-dev/lib/pkgconfig
      - /nix/store/0qp96f3rzaiiig6wafrpy0gig09gkabk-gst-editing-services-1.26.5-dev/lib/pkgconfig

  HINT: you may need to install a package such as gtk4-x11, gtk4-x11-dev or gtk4-x11-devel.

warning: build failed, waiting for other jobs to finish...
FAILED: [code=101] src/footage 
/nix/store/f8q9yh6ry4pz5z1hf8sw0ah3gznqs8sr-coreutils-9.8/bin/env CARGO_HOME=/nix/build/nix-21501-3440856152/source/build/cargo-home /nix/store/vc9ij5iq2ga02875g3i8dxz7mpa11ygf-cargo-1.91.1/bin/cargo build --manifest-path /nix/build/nix-21501-3440856152/source/Cargo.toml --target-dir /nix/build/nix-21501-3440856152/source/build/src --release &amp;&amp; cp src/release/footage src/footage
ninja: build stopped: subcommand failed.</pre>
</details>
</details>

---
### `aarch64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>footage</li>
    <li>muzika</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 45 packages built:</summary>
  <ul>
    <li>gst_all_1.gst-plugins-rs</li>
    <li>gst_all_1.gst-plugins-rs.dev</li>
    <li>mopidy-bandcamp (mopidyPackages.mopidy-bandcamp)</li>
    <li>mopidy-bandcamp.dist (mopidyPackages.mopidy-bandcamp.dist)</li>
    <li>mopidy-iris (mopidyPackages.mopidy-iris)</li>
    <li>mopidy-iris.dist (mopidyPackages.mopidy-iris.dist)</li>
    <li>mopidy-jellyfin (mopidyPackages.mopidy-jellyfin)</li>
    <li>mopidy-jellyfin.dist (mopidyPackages.mopidy-jellyfin.dist)</li>
    <li>mopidy-listenbrainz (mopidyPackages.mopidy-listenbrainz)</li>
    <li>mopidy-listenbrainz.dist (mopidyPackages.mopidy-listenbrainz.dist)</li>
    <li>mopidy-local (mopidyPackages.mopidy-local)</li>
    <li>mopidy-local.dist (mopidyPackages.mopidy-local.dist)</li>
    <li>mopidy-mopify (mopidyPackages.mopidy-mopify)</li>
    <li>mopidy-mopify.dist (mopidyPackages.mopidy-mopify.dist)</li>
    <li>mopidy-mpd (mopidyPackages.mopidy-mpd)</li>
    <li>mopidy-mpd.dist (mopidyPackages.mopidy-mpd.dist)</li>
    <li>mopidy-mpris (mopidyPackages.mopidy-mpris)</li>
    <li>mopidy-mpris.dist (mopidyPackages.mopidy-mpris.dist)</li>
    <li>mopidy-muse (mopidyPackages.mopidy-muse)</li>
    <li>mopidy-muse.dist (mopidyPackages.mopidy-muse.dist)</li>
    <li>mopidy-musicbox-webclient (mopidyPackages.mopidy-musicbox-webclient)</li>
    <li>mopidy-musicbox-webclient.dist (mopidyPackages.mopidy-musicbox-webclient.dist)</li>
    <li>mopidy-notify (mopidyPackages.mopidy-notify)</li>
    <li>mopidy-notify.dist (mopidyPackages.mopidy-notify.dist)</li>
    <li>mopidy-podcast (mopidyPackages.mopidy-podcast)</li>
    <li>mopidy-podcast.dist (mopidyPackages.mopidy-podcast.dist)</li>
    <li>mopidy-scrobbler (mopidyPackages.mopidy-scrobbler)</li>
    <li>mopidy-scrobbler.dist (mopidyPackages.mopidy-scrobbler.dist)</li>
    <li>mopidy-somafm (mopidyPackages.mopidy-somafm)</li>
    <li>mopidy-somafm.dist (mopidyPackages.mopidy-somafm.dist)</li>
    <li>mopidy-soundcloud (mopidyPackages.mopidy-soundcloud)</li>
    <li>mopidy-soundcloud.dist (mopidyPackages.mopidy-soundcloud.dist)</li>
    <li>mopidy-spotify (mopidyPackages.mopidy-spotify)</li>
    <li>mopidy-spotify.dist (mopidyPackages.mopidy-spotify.dist)</li>
    <li>mopidy-subidy (mopidyPackages.mopidy-subidy)</li>
    <li>mopidy-subidy.dist (mopidyPackages.mopidy-subidy.dist)</li>
    <li>mopidy-tidal (mopidyPackages.mopidy-tidal)</li>
    <li>mopidy-tidal.dist (mopidyPackages.mopidy-tidal.dist)</li>
    <li>mopidy-tunein (mopidyPackages.mopidy-tunein)</li>
    <li>mopidy-tunein.dist (mopidyPackages.mopidy-tunein.dist)</li>
    <li>mopidy-youtube (mopidyPackages.mopidy-youtube)</li>
    <li>mopidy-youtube.dist (mopidyPackages.mopidy-youtube.dist)</li>
    <li>mopidy-ytmusic (mopidyPackages.mopidy-ytmusic)</li>
    <li>mopidy-ytmusic.dist (mopidyPackages.mopidy-ytmusic.dist)</li>
    <li>showtime</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>footage</summary>
<pre>      - /nix/store/n0v3jmcz157j9vc6ndiwwnyfh584nhik-libadwaita-1.8.1-dev/lib/pkgconfig
      - /nix/store/ibdxk40v313qdyb4dnsgvr31xbygw0xq-a52dec-0.8.0/lib/pkgconfig
      - /nix/store/26pgxminmqj4440fczwcn7kn7jnvy3pr-fdk-aac-2.0.3-dev/lib/pkgconfig
      - /nix/store/lmvr3707jh523597vq6daaaxm2w4bdnm-ffmpeg-8.0-dev/lib/pkgconfig
      - /nix/store/x9lv91c2rcjq6ich7rzz505yij349agi-x264-0-unstable-2025-01-03-dev/lib/pkgconfig
      - /nix/store/gkcddajkvmn0baarjqy32qj3gsl386bp-x265-4.1-dev/lib/pkgconfig
      - /nix/store/r7dsy89693xr54siridvdp17b434ij46-vo-aacenc-0.1.3/lib/pkgconfig
      - /nix/store/n51niynkjgr44cbz64f1id81cmn7l6va-svt-av1-3.1.2/lib/pkgconfig
      - /nix/store/rf94r2zmz6cvmmbkj95pi1gfjq4hxlky-libmpeg2-0.5.1/lib/pkgconfig
      - /nix/store/cwblf20gf22n7hmrgl15dg78r11nffnr-gst-plugins-base-1.26.5-dev/lib/pkgconfig
      - /nix/store/7w1ff2cb8gga6pkx3zrxxmcwxw1iil83-gstreamer-1.26.5-dev/lib/pkgconfig
      - /nix/store/fshrypq9r5h16ngd18sni4w3ncvxq9g5-gst-plugins-bad-1.26.5-dev/lib/pkgconfig
      - /nix/store/8rzdzap4c32gnvjyqfb948g52yn5kmd7-gst-editing-services-1.26.5-dev/lib/pkgconfig

  HINT: you may need to install a package such as gtk4-x11, gtk4-x11-dev or gtk4-x11-devel.

warning: build failed, waiting for other jobs to finish...
FAILED: [code=101] src/footage 
/nix/store/1swaqmkr1329q50ky497sps80p16fn95-coreutils-9.8/bin/env CARGO_HOME=/nix/build/nix-12951-1114641391/source/build/cargo-home /nix/store/45b258b3av4kwr5pljhaghpkn2b076gp-cargo-1.91.1/bin/cargo build --manifest-path /nix/build/nix-12951-1114641391/source/Cargo.toml --target-dir /nix/build/nix-12951-1114641391/source/build/src --release &amp;&amp; cp src/release/footage src/footage
ninja: build stopped: subcommand failed.</pre>
</details>
</details>

--- a/pkgs/development/libraries/gstreamer/rs/default.nix
+++ b/pkgs/development/libraries/gstreamer/rs/default.nix
@@ -227,6 +227,10 @@ stdenv.mkDerivation (finalAttrs: {
     export XDG_CACHE_HOME=$(mktemp -d)
   '';
 
+  postInstall = ''
+    install -Dm444 -t ''${!outputDev}/lib/pkgconfig gst*.pc
+  '';
+
   doInstallCheck =
     (lib.elem "webp" selectedPlugins) && !stdenv.hostPlatform.isStatic && stdenv.hostPlatform.isElf;
   installCheckPhase = ''

--- a/pkgs/development/libraries/gstreamer/rs/default.nix
+++ b/pkgs/development/libraries/gstreamer/rs/default.nix
@@ -135,7 +135,7 @@ assert lib.assertMsg (invalidPlugins == [ ])
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "gst-plugins-rs";
-  version = "0.14.2";
+  version = "0.14.4";
 
   outputs = [
     "out"
@@ -147,13 +147,13 @@ stdenv.mkDerivation (finalAttrs: {
     owner = "gstreamer";
     repo = "gst-plugins-rs";
     rev = finalAttrs.version;
-    hash = "sha256-mIq8Fo6KoxAo1cL2NQHnSMPgzUWl1eNJUujdaerGjFA=";
+    hash = "sha256-MZyYHMq6gFJkVxlrmeXUjOmRYsQBHj0848cnF+7mtbU=";
   };
 
   cargoDeps = rustPlatform.fetchCargoVendor {
     inherit (finalAttrs) src;
     name = "gst-plugins-rs-${finalAttrs.version}";
-    hash = "sha256-Z1mqpVL2SES1v0flykOwoDX2/apZHxg7eI5If4BsP4o=";
+    hash = "sha256-T+fdu+Oe07Uf1YoRGYl2DMb1QgdSZVLwcOqH4bBNGXU=";
   };
 
   strictDeps = true;

